GraphsV2/Table/index.js

```diff
@@ -1085,7 +1079,4 @@ Table.propTypes = {
     data: PropTypes.arrayOf(PropTypes.object),
 };
 
-export default compose(
-    WithValidationHOC(),
```

Comment:
why do we remove validation HOC?

Reply:
https://github.com/nuagenetworks/vis-graphs/blob/3b2fcdf9f88fede97e7ea97585e531e5cea25634/HOC/WithValidationHOC.js#L21

Actually, if we don't have data then this will not render the table component, and therefore searching, etc will not be visible.